edi_bank_statement_import/tests/edi_bank_statement_import_test.py

```diff
@@ -0,0 +1,161 @@
+                    temp.seek(0)
+                    diff = list(
+                        difflib.unified_diff(
+                            golden.readlines(), temp.readlines(), golden.name, temp.name
+                        )
+                    )
+                    if len(diff) > 2:
+                        self.fail(
+                            "actual output doesn't match expected "
+                            + "output:\n%s" % "".join(diff)
+                        )
+
+    def test_parse(self):
+        self._do_parse_test("test-camt053", "golden-camt053.pydata")
+
+    def test_parse_txdtls(self):
+        self._do_parse_test("test-camt053-txdtls", "golden-camt053-txdtls.pydata")
+
+    def test_parse_no_ntry(self):
+        self._do_parse_test("test-camt053-no-ntry", "golden-camt053-no-ntry.pydata")
+
+
+class TestImport(TransactionCase):
+    """Run test to import camt import."""
+
+    transactions = [
+        {
+            "account_number": "NL46ABNA0499998748",
+            "amount": -754.25,
+            "date": date(year=2014, month=1, day=5),
+            "ref": "435005714488-ABNO33052620",
+        },
+        {
+            "remote_account": "NL46ABNA0499998748",
+            "transferred_amount": -564.05,
+            "value_date": date(year=2014, month=1, day=5),
+            "ref": "TESTBANK/NL/20141229/01206408",
+        },
+        {
+            "remote_account": "NL46ABNA0499998748",
+            "transferred_amount": -100.0,
+            "value_date": date(year=2014, month=1, day=5),
+            "ref": "TESTBANK/NL/20141229/01206407",
+        },
+        {
+            "remote_account": "NL69ABNA0522123643",
+            "transferred_amount": 1405.31,
+            "value_date": date(year=2014, month=1, day=5),
+            "ref": "115",
+        },
+    ]
+
+    def setUp(self):
+        super(TestImport, self).setUp()
+        bank = self.env["res.partner.bank"].create(
+            {
+                "acc_number": "NL77ABNA0574908765",
+                "partner_id": self.env.ref("base.main_partner").id,
+                "company_id": self.env.ref("base.main_company").id,
+                "bank_id": self.env.ref("base.res_bank_1").id,
+            }
+        )
+        self.env["res.partner.bank"].create(
+            {
+                "acc_number": "NL46ABNA0499998748",
+                "partner_id": self.env.ref("base.main_partner").id,
+                "company_id": self.env.ref("base.main_company").id,
+                "bank_id": self.env.ref("base.res_bank_1").id,
+            }
+        )
+        self.env["account.journal"].create(
+            {
+                "name": "Bank Journal - (test camt)",
+                "code": "TBNKCAMT",
+                "type": "bank",
+                "bank_account_id": bank.id,
+                "currency_id": self.env.ref("base.EUR").id,
+            }
+        )
+
+    def test_statement_import(self):
+        """Test correct creation of single statement."""
```

Comment:
here you are testing the base wizard, this should be done in the base module not here unless you are testing a special behavior provided by this new module.

Reply:
removed the tests